### ISSUE TITLE
2020-04-22: gather status update for Fedora Council 

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#450]
The Fedora Council has a status reporting system in place for the various Editions to raise awareness of what is going on in each one. We should compile some new information that we want to publicize once a month and share it by updating the https://pagure.io/Fedora-Council/status_reports/blob/master/f/modules/dashboard/pages/editions/fcos.adoc file.

The next one will be done in the weekly meeting on 2020-04-22

**Comment by dustymabe** [comment]
We discussed this in the [community meeting today](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/message/LUVFUJEZAEJOCEDYBX6RDQ7IN5CQZ6VI/).

Here are the things we've come up with that were signifcant in recent history:

- We are now performing image uploads of Fedora CoreOS into Google Cloud Platform.
- We are now producing DigitalOcean disk images that can be used with the custom image upload option.
- We are now using NetworkManager for all networking, including initramfs networking.
- We have deployed our `next` stream, based on Fedora 32 content.
- We are now delivering "metal4k" artifacts for use on disks with a 4k sector size.
- Fedora CoreOS is now offered as an option in the web UI for the Exoscale cloud provider.
